apps/prairielearn/elements/pl-big-o-input/pl-big-o-input.py

```diff
@@ -135,13 +143,18 @@ def render(element_html: str, data: pl.QuestionData) -> str:
     a_sub = None
 
     if parse_error is None and name in data["submitted_answers"]:
-        a_sub = sympy.latex(
-            psu.convert_string_to_sympy(
-                data["submitted_answers"][name],
-                variables,
-                allow_complex=False,
-                allow_trig_functions=False,
+        a_sub = data["submitted_answers"][name]
+        a_sub = (
+            sympy.latex(
+                psu.convert_string_to_sympy(
+                    a_sub,
+                    variables,
+                    allow_complex=False,
+                    allow_trig_functions=False,
+                )
             )
+            if a_sub != ""
```

Comment:
Maybe just use an if statement with an indented block instead of the inline style here? This is a little hard to parse because the function call is really long.